Markdown/MultiMarkdown.sublime-syntax

```diff
@@ -2,8 +2,9 @@
 ---
 name: MultiMarkdown
 scope: text.html.markdown.multimarkdown
+version: 2
 
-extends: Packages/Markdown/Markdown.sublime-syntax
```

Comment:
not sure why you went from explicit to implicit here

Reply:
This change follows JSX / TSX syntaxes. Relative imports work on package level only. Hence omitting path part doesn't change anything.